### PR TITLE
Userdata implemented to install and start tomcat

### PR DESCRIPTION
**Pull request by vikrant-subhash**
User data functionality used to configure and start tomcat. Also, Added ingress group rule on public security group to allow http port 80, so as to access Tomcat URL

### CODE REVIEW

File: instances.tf [hunk 1]
```diff
@@ -22,6 +22,8 @@ resource "aws_instance" "public" {
   vpc_security_group_ids      = [aws_security_group.public.id]
   subnet_id                   = aws_subnet.public[1].id
 
+  user_data = file("user_data.sh") # it is used so that instance can be startup with configuration mentioned in the script
```
Comment (vladimirmukhin) [inline instances.tf:25]: Please remove comment - it's obvious and not required

File: instances.tf [hunk 2]
```diff
@@ -40,6 +42,14 @@ resource "aws_security_group" "public" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  ingress { #Added http port to enable tomcat to access through url using port public address on port 80
```
Comment (vladimirmukhin) [inline instances.tf:45]: Please remove comment it-s obvious and not required

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Comment by vikrant-subhash** [comment]
comments not required

**Review by vladimirmukhin — APPROVED** [review]

**Comment by vikrant-subhash** [comment]
Usr Data merging

**Pull request merged**